conf/modules.config

```diff
@@ -141,6 +141,7 @@ process {
     */
 
     withName: "^.*:JOINT_GENOTYPING:MERGE_BEDS\$" {
+        ext.args = "-d 10000"
```

Comment:
make param

Reply:
done! `--merge_distance <integer>` with default of 10000